R/check_description.R

```diff
@@ -49,39 +50,40 @@ check_description <- function(x = ".") {
     !grepl("^[0-9]+\\.[0-9]+(\\.[0-9]+)?$", version)
   ] -> desc_error
   notes <- character(0)
-  if (length(commits(repo)) > 1) {
-    branch_sha <- vapply(branches(repo, "all"), branch_target, character(1))
-    head_sha <- sha(repository_head(repo))
-    current_branch <- head(names(which(branch_sha == head_sha)), 1)
+  if (length(gert::git_log(ref = gert::git_branch(repo = repo),
```

Comment:
Don't use the `package::function` notation. Unless really need functions from multiple package with the same name.

Reply:
Done in 1563e1e